param/parameterized.py

```diff
@@ -1529,8 +1677,9 @@ def override_initialization(self_,*args,**kw):
         parameterized_instance = self_.self
         original_initialized = parameterized_instance._param__private.initialized
         parameterized_instance._param__private.initialized = False
-        fn(self_, *args, **kw)
+        ret = fn(self_, *args, **kw)
```

Comment:
What if fn errors here? Should we try/except?

Reply:
No, if it errors that should be raised, could try/except to remove one level from the stack trace but that seems hairy.